api/public/cerbos/svc/v1/svc.proto

```diff
@@ -154,9 +154,25 @@ service CerbosAdminService {
         };
     }
 
+    rpc ListSchemas(cerbos.request.v1.ListSchemasRequest) returns (cerbos.response.v1.ListSchemasResponse) {
+        option (google.api.http) = {
+            post: "/admin/schema/list",
+            body: "*"
+        };
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            summary: "List schemas",
+            security: {
+                security_requirement: {
+                    key: "BasicAuth";
+                    value: {};
+                }
+            }
+        };
+    }
+
     rpc GetSchema(cerbos.request.v1.GetSchemaRequest) returns (cerbos.response.v1.GetSchemaResponse) {
         option (google.api.http) = {
-            get: "/admin/schema"
+            get: "/admin/schema/{id}",
```

Comment:
Shall we make `id` a query parameter instead of a path variable? I am a bit concerned about the asymmetry of the API. Currently it looks like:

- `POST /admin/schema`: Add/update schema
- `POST /admin/schema/list`: List schemas 
- `GET /admin/schema/list`: Get the schema with ID "list". 
- `DELETE /admin/schema/list`: Delete schema with ID "list" 

I think it would be better as follows: 
- `POST /admin/schema?id=list`: Add/update schema with ID "list"
- `POST /admin/schema/list`: List schemas 
- `GET /admin/schema?id=list`: Get the schema with ID "list". 
- `DELETE /admin/schema?id=list`: Delete schema with ID "list" 

WDYT?

Reply:
I think this looks much better, thanks for the suggestion!